assets/javascripts/discourse/templates/components/chat-draft-channel-screen.hbs

```diff
@@ -1,7 +1,31 @@
 <div class="chat-draft-channel-screen">
-  <DirectMessageCreator @onChangeSelectedUsers={{action "onChangeSelectedUsers"}} />
+  {{#if this.site.mobileView}}
+    <header class="chat-draft-channel-screen-header">
```

Comment:
@chapoi do you have a BEM recommendation for something less verbose here please?

Reply:
I think we can just cut it down to ```chat-draft``` on the top level.